lib/takeaway.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'menu'
+
+class Takeaway
+
+  attr_reader :menu_klass, :complete_order, :total_bill
+
+  def initialize(args)
+    @menu_klass     = args[:menu_klass]
+    @sendSms_klass  = args[:sendSms_klass]
```

Comment:
Style/VariableName: Use snake_case for variable names. (https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)